index.d.ts

```diff
@@ -173,6 +173,18 @@ declare const fileType: {
 	The minimum amount of bytes needed to detect a file type. Currently, it's 4100 bytes, but it can change, so don't hard-code it.
 	*/
 	readonly minimumBytes: number;
+	
+	/** 
+	Returns set of extensions supported by this library.
+	*/
+	
+	readonly extensions: object;
```

Comment:
This is not correct. It should be of type `FileType`. You also need to create a type for `mimeTypes`.

Reply:
You also need to update https://github.com/sindresorhus/file-type/blob/master/.github/pull_request_template.md